File: src/my_project/easy_problems/from251to300/epam_two_sum.py
from typing import List 

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        temp = dict()
        
        for i in range(len(nums)):
            if target - nums[i] in temp:
                return [temp[target-nums[i]], i]
            temp[nums[i]] = i
            

File: my_project_typescript/easy_problems_typescript/from151to200/epamTwoSum.ts
function twoSum(nums: number[], target: number): number[] {

    let temp: {[key: number]: number} = {}

    for (let i = 0; i < nums.length; i++){
        if (target-nums[i] in temp){
            return [temp[target-nums[i]], i]
        }
        temp[nums[i]] = i
    }
    


};